### PR TITLE
Link grant with the logged user

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: backend/api/grants/forms.py [hunk 1]
```diff
@@ -23,13 +23,23 @@ def clean(self):
         if not conference.is_grants_open:
             raise exceptions.ValidationError(_("The grants form is now closed!"))
 
+        try:
+            Grant.objects.get(user_id=self.context.request.user.id)
+            raise exceptions.ValidationError(_("Grant already submitted!"))
+        except Grant.DoesNotExist:
+            pass
+
```
Comment (marcoacierno) [inline backend/api/grants/forms.py:31]:
```
if Grant.objects.filter(user_id=self.context.request.user.id).exists()
```

this way django doesn't do an a select query fetching data/creating the object just to throw it away

File: frontend/src/pages/grants/index.tsx [hunk 2]
```diff
@@ -87,13 +92,31 @@ export const GrantsPage = () => {
           my: 5,
         }}
       >
-        {status === DeadlineStatus.HappeningNow && (
-          <GrantForm conference={code} />
-        )}
-        {status === DeadlineStatus.InTheFuture && (
-          <GrantsComingSoon start={start} />
+        {isLoggedIn ? (
```
Comment (marcoacierno) [inline frontend/src/pages/grants/index.tsx:95]: what happens if we remove this condition?

Comment (estyxx) [reply frontend/src/pages/grants/index.tsx:95]: the page already redirect! I forgot to remove this!

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**